src/openrct2/platform/Shared.cpp

```diff
@@ -94,12 +94,23 @@ namespace Platform
         return outTime;
     }
 
-    bool OriginalGameDataExists(std::string_view path)
+    bool IsRCT2ClassicPath(std::string_view path)
     {
-        std::string combinedPath = Path::ResolveCasing(Path::Combine(path, u8"Data", u8"g1.dat"));
+        auto combinedPath = Path::ResolveCasing(Path::Combine(path, u8"Assets", u8"g1.dat"));
         return File::Exists(combinedPath);
     }
 
+    bool OriginalGameDataExists(std::string_view path)
+    {
+        // Check if path is RCT2
+        auto combinedPath = Path::ResolveCasing(Path::Combine(path, u8"Data", u8"g1.dat"));
```

Comment:
Could you spin this off into its own function like the one for Classic, as well as remove the comments (they state the obvious)?